packages/calypso-e2e/src/data-helper.ts

```diff
@@ -34,16 +34,39 @@ export function getCalypsoURL(
 /**
  * Returns the credential for a specified account from the configuration file.
  *
- * @param {string} username Username of the account for which the password is to be obtained.
- * @returns {string} Credential entry found in the configuration file for the given username.
+ * @param {string} username Username of the account for which the credentials are to be obtained.
+ * @returns {string[]} Credential entry found in the configuration file for the given username.
  * @throws {Error} If username does not correspond to a valid entry in the configuration file.
  */
-export function getAccountCredential( username: string ): string {
+export function getAccountCredential( username: string ): string[] {
 	const testAccounts: { [ key: string ]: string } = config.get( 'testAccounts' );
 	try {
-		return testAccounts[ username ];
+		const [ user_name, password ] = testAccounts[ username ];
+		return [ user_name, password ];
 	} catch ( err ) {
-		throw new Error( `Credential for username: ${ username } not found in configuration file.` );
+		throw new Error(
+			`Configuration file did not contain credentials for requested user ${ username }.`
+		);
+	}
+}
+
+/**
+ * Returns the site URL for a specified account from the configuration file.
+ *
+ * @param {string} username Username of the account for which the site URL is to be obtained.
+ * @returns {string} Site URL for the given username.
+ * @throws {Error} If the username does not have a corresponding site URL, or username does not have an entry in the configuration file.
+ */
+export function getAccountSiteURL( username: string ): string {
+	const testAccounts: { [ key: string ]: string } = config.get( 'testAccounts' );
+	try {
+		const [ , , url ] = testAccounts[ username ];
+		if ( url ) {
+			return `https://${ url }`;
```

Comment:
Playwright needs the full `https://` prefix in order to navigate.

I did not want to rely on an external library like [JS-URI](https://medialize.github.io/URI.js/) just for this, especially since we control the contents of the credentials file.

Reply:
Would it make sense to use the URL constructor here?

```js
return new URL( `https://${ url }` );
```

I guess the benefit of that would be the validation of the URL string.

Reply:
> Would it make sense to use the URL constructor here?
> 
> ```js
> return new URL( `https://${ url }` );
> ```
> 
> I guess the benefit of that would be the validation of the URL string.

Yeah I came across another part of my code that used the URL constructor. It would make sense to switch to that for here.